### ISSUE TITLE
Option to Choose Secondary Color/Size

### ISSUE THREAD

**Issue by gaoe890** [val92130/Supreme-Auto-Checkout#95]
If a color and/or size is sold out, have a secondary option for auto cop instead of manually inputting. 
This would help a lot especially in the upcoming box logo season.

**Comment by Ramil2** [comment]
Yes! Can you make more than one preferred colors pllsssssssss!!! It would be awesome!

**Comment by val92130** [comment]
Hi, you can input multiple colors in an Autocop product, for example if you input "blue black" and only the color black is available then it will choose the black product :)

**Comment by Ramil2** [comment]
Thanks!!! I understood this after some tries)), but can you explain how many colors can I input, and also is there a priority(for example I input “blue black red”, so if blue sold out it will check black, and after this red?), also don’t u know, can bot be banned if I start it before drop(6-5 sec before), and I need only to fill CVV myself for correctly payment? 
Sorry for many questions and poor English)
